packages/react-components/src/components/styles/VideoTile.styles.ts

```diff
@@ -211,6 +211,6 @@ export const reactionRenderingStyle = (args: { spriteImageUrl: string; personaSi
     justifyContent: 'center',
     alignItems: 'center',
     backgroundPosition: `center`,
-    transform: `scale(${84 < args.personaSize ? 84 / args.personaSize : args.personaSize / 84})`
+    transform: `scale(${84 < args.emojiSize ? 84 / args.emojiSize : args.emojiSize / 84})`
```

Comment:
nit: can we make 84 a constant so we know why it was picked for these calculations?